CHANGELOG.md

```diff
@@ -4,6 +4,16 @@
 
 ### 🛑 Breaking changes 🛑
 
+### 🚩 Deprecations 🚩
```

Comment:
Nice, we should indeed add those sections during the release.

Reply:
It is missing the "New components" section though, isn't it?

Reply:
I guess we don't usually have New Components on the core repo? Last time we used that section on the core repo changelog was on v0.14.0

Reply:
I copied the sections that existed and deleted the content. I don't think we have new components in this release and there are no immediate plans to add new components to this repo, so probably don't need that section anyway.